lib/service/cfg.go

```diff
@@ -364,6 +365,9 @@ type ProxyConfig struct {
 	// MySQLAddr is address of MySQL proxy.
 	MySQLAddr utils.NetAddr
 
+	// RedisAddr is address of Redis proxy.
+	RedisAddr utils.NetAddr
```

Comment:
Right now the `RedisAddr` is never set.

Reply:
As @r0mant asked in this PR. I'll extract changes and create a separate PR for that.